Goobi/plugins/opac/ModsPlugin/org/kitodo/production/plugin/CataloguePlugin/ModsPlugin/UGHUtils.java

```diff
@@ -0,0 +1,249 @@
+/*
```

Comment:
As this file contains plugin specific code, this file should be stay inside plugin. There is a identical named file in PicaPlugin too.